test/executables/intl_message_migration_test.dart

```diff
@@ -23,7 +23,6 @@ import 'mui_migration_test.dart' show testCodemod;
 
 // Change this to `true` and all of the functional tests in this file will print
 // the stdout/stderr of the codemod processes.
-final _debug = false;
```

Comment:
It seems this is unused, but if we're deleting it, we should also delete the comment.

Reply:
Removed unused comment from here